### PR TITLE
feat: add wapmorgan/mp3info and id3parser

### PR DESCRIPTION
**Pull request by kesselb**
We forked getID3 to provide a library to only read MP3 files but nothing else.
Our primary use case is to use an embedded image as a preview.

### CODE REVIEW

File: wapmorgan/mp3info/_config.yml
Comment (nickvergessen) [inline wapmorgan/mp3info/_config.yml:1]: Should be git-ignored

Comment (kesselb) [reply wapmorgan/mp3info/_config.yml:1]: Well spotted :+1:

File: composer/installed.php [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <?php return array(
     'root' => array(
         'name' => 'nextcloud/3rdparty',
-        'pretty_version' => 'dev-master',
-        'version' => 'dev-master',
-        'reference' => 'b28af7ccf0bebff554672543fbbfe0e5d99968bd',
+        'pretty_version' => '1.0.0+no-version-set',
+        'version' => '1.0.0.0',
+        'reference' => NULL,
```
Comment (come-nc) [inline composer/installed.php:6]: That part looks weird, is that expected?

Comment (kesselb) [reply composer/installed.php]: Indeed. 

`COMPOSER_ROOT_VERSION=dev-master composer update wapmorgan/mp3info --no-dev` made it work.

**Review by come-nc — APPROVED** [review]

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**